### PR TITLE
Warp: Allow spaces in names, fix negative coordinates, listing filter and sort

### PR DESCRIPTION
**Pull request by ElNounch**
Partial fix for https://github.com/cuberite/cuberite/issues/3305 ( /warp )

### CODE REVIEW

**Comment by NiLSPACE** [comment]
ChunkStay was fixed, so you can keep that in: cuberite/cuberite#3319

**Comment by ElNounch** [comment]
Fix #43

File: warps.lua [hunk 1]
```diff
@@ -1,101 +1,160 @@
...
+	end
+	return Result
+end
+
+local function GetWarpByName( WarpName )
+	return warps[WarpName]
+end
+
+local function SetWarpByName( WarpName, NeoValue )
+	if (WarpsINI:FindKey(WarpName) == cIniFile.noID) then
```
Comment (ElNounch) [inline warps.lua]: Beware of https://github.com/cuberite/cuberite/issues/3324

**Comment by tonibm19** [comment]
If you've tested and it's all right I say merge.

**Comment by ElNounch** [comment]
Thank you, but currently under heavy work. Should commit soon ! :tm:

**Comment by mathiascode** [comment]
Closing this PR due to large changes in the codebase. Feel free to open a new PR if you want the changes merged.

**Pull request closed without merging**